apps/src/lib/levelbuilder/script-editor/NewScriptForm.jsx

```diff
@@ -11,7 +12,17 @@ export default function NewScriptForm() {
   return (
     <form action="/s" method="post">
       <RailsAuthenticityToken />
-      <label>Name</label>
+      <label>
+        Script Slug
+        <HelpTip>
+          <p>
+            The script slug is used to create the link to the script. It is in
+            the format of studio.code.org/s/script_slug_here. A script slug can
+            only contain lowercase letters, numbers and dashes. Once you set the
+            slug it can not be updated.
```

Comment:
since the explanation says it can only use letters, number or dashes, I think it would be helpful to use script-slug-here in the example instead of script_slug_here